lib/src/options.dart

```diff
@@ -0,0 +1,17 @@
+class Options {
```

Comment:
I thought about the circular import thing – it annoys me, but I understand.

As far as cleanup, I point to the diff in `html_generator`. 25 lines removed.

We were referencing each value 3 times – and offering no value.

Options also allows us to centralize defaults is a much more clean way.